openam-core/src/main/java/org/forgerock/openam/ldap/LDAPAuthUtils.java

```diff
@@ -1320,6 +1320,13 @@ public String getUserId(String name) {
         }
     }
 
+    /**
+     * TODO-JAVADOC
```

Comment:
Don't omit comments...
And please add a copyright header to this file.

Reply:
IC.